Dockerfile.rhel7

```diff
@@ -14,7 +14,8 @@ COPY --from=builder /tmp/build/console /usr/bin/console
 COPY manifests /manifests/
 
 # extensions manifests generated from openshift/api types
-COPY vendor/github.com/openshift/api/console/v1/*.yaml /manifests
+COPY vendor/github.com/openshift/api/console/v1/*.yaml /manifests/
+COPY vendor/github.com/openshift/api/operator/v1/0000_70_console-operator.crd.yaml /manifests/
```

Comment:
Is Dockerfile.rhel7 still used? I'm wondering if we can remove it and only maintain one Dockerfile.

Reply:
Good Q.  The `machine-`* operators, `cluster-version` operator, and `api-server` operators still have a `.rhel7` file.  I'm assuming its still in use.